config/drupal.settings

```diff
@@ -32,6 +32,11 @@ $config_directories[CONFIG_SYNC_DIRECTORY] = getenv('CONFIG_SYNC_DIRECTORY');
 // $settings['cache']['default'] = 'cache.backend.memcache';
 $settings['memcache']['servers'] = [sprintf('%s:%s', getenv('MEMCACHE_HOSTNAME'), getenv('MEMCACHE_PORT')) => 'default'];
 
+// Set config split environment.
+$config['config_split.config_split.local_development']['status'] = TRUE;
+$config['config_split.config_split.development']['status'] = FALSE;
```

Comment:
What's the difference between local_development and development ?

Reply:
local_development is effectively your local environment (Lando). I'd normally expect to call this 'dev' but that crosses over uncomfortably close to what is called the dev server (lives in the stables network) so would easily get confusing.

Happy to shorten to `local` if clearer?

Reply:
Yeah, local it clearer to me.

Reply:
Now done, cheers